### ISSUE TITLE
GNorm2 training dataset

### ISSUE THREAD

**Issue by duncster94** [ncbi/GNorm2#7]
Hi, I was wondering where I can find the training set used to train GNorm2, specifically for gene/protein NER and normalization. I would like to benchmark GNorm2 against a subset of genes (specifically yeast genes only).

Thanks!
